stix_shifter_modules/guardium/stix_transmission/guard_utils.py

```diff
@@ -113,11 +114,28 @@ def handle_report(self, report_name, params, index_from, fetch_size):
         # -------------------------------------------------------------------------------
         results = ""
         #context().logger.debug('-------------------  ' + report_name + ' ----------------------')
-        params_set = {"reportName":"{0}".format(report_name), "indexFrom": "{0}".format(index_from), "fetchSize": "{0}".format(fetch_size), "reportParameter":params}
+        params_set = {"reportName":"{0}".format(report_name), "indexFrom": "{0}".format(index_from), "fetchSize": "{0}".format(fetch_size), "reportParameter":params, "inputTZ":"GMT"}
         json_dump = json.dumps(params_set)
         rest_data = str(json.loads(json_dump))
-
-        response = requests.post(self.url+self.report_target, data=rest_data, headers=self.headers, verify=False)
+        # For compatibility with Guardium - inputTZ parameter was aded after v11.3 so in case it does not exist execute the query without it
+        for x in range(2):
+            response = requests.post(self.url+self.report_target, data=rest_data, headers=self.headers, verify=False)
+            results = response.json()
+            check_list = isinstance(results, list)
+            if check_list == False:
+                try:
+                    errorCode = results["ErrorCode"]
+                    print(errorCode)
+                    if errorCode ==  "27":
+                        params_set.pop("inputTZ")
+                        json_dump = json.dumps(params_set)
+                        rest_data = str(json.loads(json_dump))
+                        self.logger.warn("InputTZ not suppoerted - running query without it")
+                        print("InputTZ not suppoerted - running query without it")
+                        continue
+                except:
```

Comment:
not sure what's the point of having this try/except if you are not throwing any exception.

Reply:
I am not sure if exceptions work the same way in Python - in Java this would prevent the exception from being thrown upwards.